utils/fetchEarthquakes.ts

```diff
@@ -26,9 +26,29 @@ export interface EventTimeAndMagnitude {
   magnitude: number;
 }
 
-// Attempt at single function to handle all day needs
 export const fetchDailyStats = async (): Promise<any> => {
-  // fetch the earth quakes for the day
+  const res = await axios.get(
+    'https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_day.geojson'
+  );
+
+  const dailyEvents = res.data.features;
+
+  // sort by time and magnitude - for chart
+  const dailyEventsWithTimes = dailyEvents.map(
+    (feature: EarthquakeFeature) => ({
+      time: feature.properties?.time,
+      magnitude: feature.properties?.mag,
+    })
+  );
+
+  return {
+    dailyEvents,
+    dailyEventsWithTimes,
+  };
+};
+
+export const fetchDailyGraphStats = async (): Promise<any> => {
```

Comment:
This util file used to be loaded with one-off functions calling the usgs API in different ways and we cleaned it up to bring it down to 2. This adds a slightly different daily events fetch that shouldn't really need to exist. 

It might be nice to create a class, service, etc to handle fetching whatever data we're looking for.